### PR TITLE
(#4575) - Use existing COUCH_HOST when defined

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by daleharvey** [comment]
Bonus it didnt break much :) So now the browser couch tests are failing due to cors, not sure if the node tests are intermittent of there is another issue, but getting closer

**Comment by daleharvey** [comment]
@garrensmith Hey, whats the easiest way to configure cors on 2.0? I would really like either a tool or an api that does not require the user (me) to care about the cluster configuration / node names etc, it just knows couchdb is at http://120.0.0.1:3001 and turns cors on

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ before_install:
   - docker run -d -p 3001:5984 klaemo/couchdb:2.0-dev --with-haproxy --with-admin-party-please -n 1
 
   # Because Saucelabs doesnt proxy 5984 on OSX
-  - export COUCH_HOST=http://127.0.0.1:3000
+  - "if [ -z \"$COUCH_HOST\" ]; then export COUCH_HOST=http://127.0.0.1:3000; fi"
```
Comment (kxepal) [inline .travis.yml]: `export COUCH_HOST=${COUCH_HOST:-http://127.0.0.1:3000}` if how to use default when variable is not defined.

**Comment by kxepal** [comment]
@daleharvey So far this is the way to go: https://github.com/klaemo/docker-couchdb/issues/42#issuecomment-169610897

**Comment by garrensmith** [comment]
@robertkowalski and I have been working on NMO a command line client. It should be able to do the job. I'll try take a look this week. 

All misspelling thanks to my iPhone

**Comment by nolanlawson** [comment]
The couch-master tests are still throwing due to this:

```
 1) test.sync.js-http-local sync throws errors in error listener:
     missing
```

But this seems to get us one step closer, so I'm :+1:.

**Comment by nolanlawson** [comment]
21e1346e11e611f1d312f468a711389038b04287

**Pull request closed without merging**